portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Comments</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body onload="getData()">
+    <div id="content">
+      <h1>Comments:</h1>
+
+      <!-- Comments Data -->
+      <ul id="data-container"></ul>
+
+      <br></br>
+
+      <!-- Leave a comment -->
+      <form action="/data" method="POST">
+        <p style="margin-top:100px">Leave a comment on my page!</p>
+        <textarea name="text-input">Hi Alan!</textarea>
```

Comment:
I'd consider doing <textarea name="text-input" placeholder="Hi Alan!"></textarea> instead, so that you don't have to delete the whole comment if you want to type something other than "Hi Alan!"